.vscode/extensions.json

```diff
@@ -1,6 +1,8 @@
 {
     "recommendations": [
         "golang.go",
-        "maxnatchanon.go-struct-tag-autogen"
+        "maxnatchanon.go-struct-tag-autogen",
+        // Not needed, but nice for highlighting syntax & such
```

Comment:
JSON does not support comments

Reply:
wait then how come launch.json has comments???

Reply:
![image](https://github.com/user-attachments/assets/ebc4fcf0-571b-4843-a9d0-5a129fe6fde7)